install/kubernetes/cilium/values.yaml

```diff
@@ -1338,6 +1338,18 @@ monitor:
   # endpoints filtering
   # serviceTopology: false
 
+  # -- LoadBalancer with envoy proxy
+  envoy:
+    # -- Enable L7 service load balancing via envoy proxy.
+    # The request to a k8s service, which has specific annotation e.g. io.cilium/lb-protocol,
+    # will be forwarded to the local envoy proxy to be load balanced to the service endpoints.
+    # Please refer to docs for supported annotations for configuring the envoy proxy.
+    # This will automatically set enable-envoy-config as well.
+    enabled: false
+    # -- List of ports from service to be automatically redirected to envoy proxy.
```

Comment:
Does this apply to all services, or only those with the annotation?

Reply:
this will be applied for all services, good spot :eyes:. I will update comment to make it clearer.

Reply:
(we discussed out-of-band, just need to make this clear that this applies to **all services** in the cluster)